sdk/openai/Azure.AI.OpenAI/src/Generated/Models/Choice.cs

```diff
@@ -5,7 +5,9 @@
 
 #nullable disable
 
-namespace Azure.AI.OpenAI
+using Azure.AI.OpenAI;
+
+namespace Azure.OpenAI.Models
```

Comment:
Not sure why 42 is causing this change but it might be something we need to tweak in our emitter / tcgc to prevent.